README.md

```diff
@@ -7,101 +7,39 @@ has an identity layer (Azimuth), virtual machine (Vere), and operating system
 A running Urbit "ship" is designed to operate with other ships peer-to-peer.
 Urbit is a general-purpose, peer-to-peer computer and network.
 
-This repository contains:
+This repository contains the [Arvo Kernel][arvo]
 
-- The [Arvo OS][arvo]
-- [herb][herb], a tool for Unix control of an Urbit ship
-- Source code for [Landscape's web interface][land]
-- Source code for the [vere][vere] virtual machine.
-
-For more on the identity layer, see [Azimuth][azim]. To manage your Urbit
-identity, use [Bridge][brid].
-
-[arvo]: https://github.com/urbit/urbit/tree/master/pkg/arvo
-[azim]: https://github.com/urbit/azimuth
-[brid]: https://github.com/urbit/bridge
-[herb]: https://github.com/urbit/urbit/tree/master/pkg/herb
-[land]: https://github.com/urbit/urbit/tree/master/pkg/interface
-[vere]: https://github.com/urbit/urbit/tree/master/pkg/urbit
+For the Runtime, see [Vere][vere].
+For Landscape (the UI), see [Garden][gard].
```

Comment:
There's also a question of whether to redirect to Groups/`tloncorp` repos here as well, particularly given the equivocation around the name "Landscape" over the years.

Reply:
And I assume we should make reciprocal links from those...

Reply:
Yeah - Josh asked to just remove those so I did